cryptoxlib/clients/bitpanda/BitpandaWebsocket.py

```diff
@@ -172,3 +216,90 @@ def get_subscription_message(self, **kwargs) -> dict:
             "name": self.get_channel_name(),
             "instrument_codes": map_multiple_pairs(self.pairs, sort = True)
         }
+
+
+class TradingSubscription(BitpandaSubscription):
+    def __init__(self, callbacks: CallbacksType = None):
+        super().__init__(callbacks)
+
+    @staticmethod
+    def get_channel_name():
+        return "TRADING"
+
+    def get_subscription_message(self, **kwargs) -> dict:
+        return {
+            "name": self.get_channel_name(),
+        }
+
+    def requires_authentication(self) -> bool:
+        return True
+
+
+class OrdersSubscription(BitpandaSubscription):
+    def __init__(self, callbacks: CallbacksType = None):
+        super().__init__(callbacks)
+
+    @staticmethod
+    def get_channel_name():
+        return "ORDERS"
+
+    def get_subscription_message(self, **kwargs) -> dict:
+        return {
+            "name": self.get_channel_name(),
+        }
+
+    def requires_authentication(self) -> bool:
+        return True
+
+
+class CreateOrderMessage(WebsocketOutboundMessage):
+    def __init__(self, pair: Pair, type: enums.OrderType, side: enums.OrderSide, amount: str, price: str = None,
+                 stop_price: str = None, client_id: str = None):
+        self.pair = pair
+        self.type = type
+        self.side = side
+        self.amount = amount
+        self.price = price
+        self.stop_price = stop_price
+        self.client_id = client_id
```

Comment:
Bitpanda Pro added more order variants via the `time_in_force` parameter, e.g. FILL_OR_KILL .

see https://developers.bitpanda.com/exchange/#properties-11

I could provide a pull request too if you do not want to add it in this one.

Reply:
Good to know, will be added to the PR, thanks.